src/test/resources/messages.properties

```diff
@@ -9,6 +9,9 @@ invalid.past-or-present.date=${validatedValue} date couldn't be in the future
 invalid.password=Your password must be between 8 and 30 characters at must contain at least: 1 uppercase Latin character (A-Z), 1 lowercase Latin character (a-z), 1 digit (0-9)
 invalid.username.size=Your login must be between 2 and 50 characters, do not use space
 invalid.token=You must be authenticated
+token.not.start.with.bearer=JWT token is not starts with Bearer prefix
+token.signature.not.valid=JWT signature does not match locally computed signature. JWT validity cannot be asserted and should not be trusted.
```

Comment:
Why do you verifying this message into the tests? The user won't see it on production env?

Reply:
This message will be received in the detailed authentication error message in response 
if the token signature is violated.